src/components/Dashboard/CustodianListSm.vue

```diff
@@ -67,11 +67,19 @@ import { Component, Mixins, Prop } from 'vue-property-decorator'
 
 // Mixins
 import { CommonMixin, CountriesProvincesMixin } from '@/mixins'
+import { PartiesIF } from '@/interfaces'
 
 @Component({})
 export default class CustodianListSm extends Mixins(CommonMixin, CountriesProvincesMixin) {
-  @Prop({ default: [] })
-  readonly custodians: Array<object>
+  @Prop({ default: () => [] })
+  readonly custodians: Array<PartiesIF>
+
+  /** Return the officer person or org name */
+  private officerName (party: PartiesIF): string {
+    return party.officer.firstName
+      ? `${party.officer.firstName} ${party.officer.lastName}`
```

Comment:
Any thoughts about whether middle initial should be in here, too? I think there's an outstanding ticket for that.

Reply:
Good question, this is sort of a port from DirectorsListSm template. 
Is the middle initial something we want to include for both Directors and (Person) Custodians?

Reply:
Search for that other ticket. If you can't find it then the answer is no. Otherwise you could kill 2 🐦 with one change.

Reply:
I found ticket#: 10452, but it's pretty vague. Either way, seems like there is a desire to include middle names, in this case the middle initial. Will try to fit that in today!

Reply:
Put the middle name into the directors accordion as well as custodians accordion. 
1 snag though, currently the custodians party object is only returning first and last name, when i explicitly put it into the filing. It might not be passed through at some point in the Dissolutions flow perhaps?
Either way, something to look for there, but this UI will support/display it once we have that value in the response.